rclcpp_generic/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rclcpp_generic</name>
+  <version>0.1.0</version>
+  <description>Communication between nodes with the message type chosen at runtime</description>
+  <maintainer email="nikolai.morin@apex.ai">Nikolai Morin</maintainer>
```

Comment:
Should I keep the `rosbag2_transport` maintainers here? Or maybe some working group?

Reply:
It may be appropriate to add the maintainers from the other `rclcpp` packages - given that it lives as a group, a repository usually shares the same maintainers across its packages. It would be good to run it by them first.

Reply:
Okay, I asked the `rclcpp` maintainers.